src/multi.jl

```diff
@@ -40,85 +100,93 @@ function amp_convexify_multilinear(m::PODNonlinearModel, k, λ::Dict, α::Dict,
     return λ, α
 end
 
-function amp_convexify_monomial(m::PODNonlinearModel, k, λ::Dict, α::Dict, discretization::Dict)
-    m.nonlinear_terms[k][:convexified] = true  # Bookeeping the convexified terms
+function amp_convexify_intprod(m::PODNonlinearModel, k::Any, λ::Dict, α::Dict, d::Dict)
```

Comment:
Are these functions well tested

Reply:
All integer problems are less tested as they are currently experimental. I have a few more tests written in the next PR but not extensively.